chapters/hpmor-chapter-013.tex

```diff
@@ -150,15 +150,34 @@ \chapter{Asking the Wrong Questions}
 
 Harry shrugged despairingly and went over to the cabinet he’d been given near his bed, to get his wizard’s robes for the day.
 
-On the floor of the cabinet, under his robes, were the cereal bars, and a note: \begin{align*} \hbox{\scshape Points spent: }&\hbox{1}\\ \hbox{\scshape Current points: }&\hbox{97}\\ \hbox{\scshape You have gained: }&\hbox{\scshape 6 cereal bars}\\ \hbox{\scshape You are still wearing: }&\hbox{\scshape Pyjamas}\\[1.5ex] \multicolumn{2}{c}{\hbox{\scshape Do not eat while you are wearing your pyjamas}}\\ \multicolumn{2}{c}{\scshape You will get a Pyjama Penalty} \end{align*}
+On the floor of the cabinet, under his robes, were the cereal bars, and a note:
+\begin{align*} \hbox{\scshape Points spent: }&\hbox{1}\\
+\hbox{\scshape Current points: }&\hbox{97}\\
+\hbox{\scshape You have gained: }&\hbox{\scshape 6 cereal bars}\\
+\hbox{\scshape You are still wearing: }&\hbox{\scshape Pyjamas}\\%
+[1.5ex] \multicolumn{2}{c}{\hbox{\scshape Do not eat while you are wearing your pyjamas}}\\
+\multicolumn{2}{c}{\scshape You will get a Pyjama Penalty}
+\end{align*}
 
 \emph{And now I know that whoever controls the game is insane.}
 
 “My guess is that the game is controlled by Dumbledore,” Harry said out loud. Maybe \emph{this} time he could set a new land speed record for being quick on the uptake.
 
 Silence.
 
-But Harry was starting to pick up the pattern; the note would be in the next place he looked. So Harry looked under his bed. \begin{align*}\intertext{\scshape \centering Ha! Ha ha ha ha ha!\\ Ha ha ha ha ha ha!\\ Ha! Ha! Ha! Ha! Ha! Ha!\\ Dumbledore does not control the game\\ Bad guess\\ Very bad guess\\ −20 points\\ And you are still wearing pyjamas\\ it is your fourth move\\ and you are still wearing pyjamas} \hbox{\scshape Pyjama penalty: }&\hbox{\scshape −2 points}\\ \hbox{\scshape Current points: }&\hbox{75} \end{align*}
+But Harry was starting to pick up the pattern; the note would be in the next place he looked. So Harry looked under his bed.
+\begin{align*}\intertext{\scshape \centering Ha! Ha ha ha ha ha!\\
```

Comment:
Again, please have a line break after the start of the environment.

Reply:
fixed